src/Discovery/DiscoveryListView.tsx

```diff
@@ -47,15 +47,22 @@ const DiscoveryListView: React.FunctionComponent<Props> = (props: Props) => {
       columns={props.columns}
       rowKey={props.config.minimalFieldMapping.uid}
       rowSelection={(
-        props.config.features.exportToWorkspace
-              && props.config.features.exportToWorkspace.enabled
+        (props.config.features.exportToWorkspace
+          && props.config.features.exportToWorkspace.enabled) || (props.config.features.exportToWorkspace?.enableFillRequestForm
+            && props.config.features.exportToWorkspace.enableFillRequestForm === true)
       ) && {
         selectedRowKeys: props.selectedResources.map(
           (r) => r[props.config.minimalFieldMapping.uid],
         ),
         renderCell: (_checked, _record, _index, node) => (
           <Tooltip
-            title={`Click to select item for ${(props.config.features.exportToWorkspace.enableDownloadManifest || props.config.features.exportToWorkspace.enableDownloadZip) ? 'download or ' : ''}open in workspace`}
+            title={`Click to select item for ${
+              props.config.features.exportToWorkspace.enableFillRequestForm
+                ? props.config.features.exportToWorkspace.fillRequestFormDisplayText.toLowerCase()
+                : (props.config.features.exportToWorkspace.enableDownloadManifest || props.config.features.exportToWorkspace.enableDownloadZip)
+                  ? 'download or open in workspace'
+                  : 'open in workspace'
+              }`}
```

Comment:
Actually reading this again, this is making to checkbox tooltip texts inaccurate when both the fill request form button and export/download button are enabled right?

Reply:
Thanks for pointing that out! I have updated the tooltip to clearly list all enabled options when multiple actions (like "Fill Request Form" and "Download") are available.